jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/GridListTests.java

```diff
@@ -0,0 +1,54 @@
+package io.github.epam.angular.tests.elements.complex;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.AngularPage.gridListSection;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+
+public class GridListTests extends TestsInit {
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        angularPage.shouldBeOpened();
+    }
+
+    @Test
+    public void basicGridListBasicTest() {
+        gridListSection.basicGridList.is().displayed();
```

Comment:
Can you explain the logic of the test?
Usually, first preparation then verification of the result, but here it seems to me the opposite.